apollo/lib/security/__init__.py

```diff
@@ -56,8 +57,13 @@ def get_principals(enhanced_http_connection, session):
 
         authenticated_user = enhanced_http_connection.current_user
         if authenticated_user:
-            principals += [Authenticated, Human,
-                           f"user:{authenticated_user.id}"]
+            principals += [Human, f"user:{authenticated_user.id}"]
```

Comment:
I'm thinking we'd have to move both these principals into the case for when the user has been initialised as well. Now we'd inevitably get issues where `Uninitialized` users are able to call endpoints covered by `Human` and the user specific principal.

I don't think the principal naming is perfect at that point though, specifically `Human` so I'll think about a more logical set of principals covering the relevant cases.